src/Common/src/Interop/OSX/Interop.CoreFoundation.CFDate.cs

```diff
@@ -8,7 +8,8 @@
 
 using Microsoft.Win32.SafeHandles;
 
-using CFAbsoluteTime=System.Double;
+#pragma warning disable SA1121 // we don't want to simplify built-ins here as we're using aliasing
```

Comment:
📝 I'm considering allowing this case across the board for SA1121, or at least adding an option for it to **stylecop.json**